src/robot_gnss/README.md

```diff
@@ -0,0 +1,36 @@
+# Поддержка модуля GNSS
```

Comment:
Edit your readme.md file. Translate all Russian text in English, please.

Reply:
Fixed